Algebras/Shapes/alg_shapes_rope.gap

```diff
@@ -1,303 +1,12 @@
-algebra alg_shapeX implements sig_foldrna(alphabet = char, answer = Rope) {
```

Comment:
Thanks for splitting this file. However, for really productive system we need to implement the Shape algebra with a more efficient data type "shape_t". Have a look at `https://github.com/jlab/fold-grammars/blob/master/Algebras/Shapes/Parts/algpart_shapeX_basic.gap`, but there are some technical issues with the underlying header files in bgap, thus this has to wait. Ideally, we would have ONE algebra for cofolding for all 5 shape levels --> one binary instead of 5